workflow/report/loadings-plot.rst

```diff
@@ -1 +1,2 @@
-Shows principal components of {{ snakemake.wildcards.covariate }}, which transcripts contribute the most.
+**Plot loadings**, computed with sleuth, shows for each principal components of {{ snakemake.wildcards.covariate }}, which transcripts (estimated counts) contribute the most.
```

Comment:
Not sure if this is needed, but for consistency I would add the leading and trailing double \`\` before and after the jinja template string (i.e. before and after the `{{}}`)